src/main/java/org/usfirst/frc/team2473/robot/Robot.java

```diff
@@ -103,7 +101,10 @@ public void autonomousInit() {
 	 * Runs continuously during the autonomous state
 	 */
 	@Override
-	public void autonomousPeriodic() {		
+	public void autonomousPeriodic() {
+        Devices.getInstance().updateVisionValues();	
```

Comment:
Continuing on the theme of modularity, can the task of updating from serial be a command? It's possible the answer is no, but we should think about what we are letting bypass the Scheduler.

Reply:
According to WPILib: `Commands - define the operation of the robot incorporating the capabilities defined in the subsystems.`

I feel like this should stay as not a command since it's just an update of a variable.